README.md

```diff
@@ -3,11 +3,758 @@
 [![Build
 Status](https://ci.jenkins.io/job/Plugins/job/cobertura-plugin/job/master/badge/icon)](https://ci.jenkins.io/job/Plugins/job/cobertura-plugin/job/master/)
 
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/cobertura.svg)](https://plugins.jenkins.io/cobertura)
+
+<!-- Commenting out the changelog badge as >
+<!-- [![GitHub release](https://img.shields.io/github/release/jenkinsci/cobertura.svg?label=changelog)](https://github.com/jenkinsci/cobertura-plugin/releases/latest) -->
+
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/cobertura.svg?color=blue)](https://plugins.jenkins.io/cobertura)
+
 This plugin allows you to capture code coverage report from
-[Cobertura](https://github.com/cobertura/cobertura/).
-Jenkins will generate the trend report of coverage.
-The Cobertura plugin can be [downloaded
+[Cobertura](http://cobertura.sourceforge.net/). Jenkins will generate
+the trend report of coverage.The Cobertura plugin can be [downloaded
 here](http://updates.jenkins-ci.org/download/plugins/cobertura/).
 
-For more information, see [the Wiki
-page](https://wiki.jenkins-ci.org/display/JENKINS/Cobertura+Plugin).
+## Version History
+
+- Version history can be found below: [CHANGELOG](#changelog)
+- Tagged releases can be found on the following page: [Releases](https://github.com/dduportal/cobertura-plugin/releases)
```

Comment:
https://github.com/jenkinsci/cobertura-plugin/releases

Reply:
Good catch, I'm sorry for the bad copy and paste